File: patterns/javascript-patterns/pattern64.js
// Pattern 64: ...

// 0
// 1*
// 2**
// 3***
// 4****
// 5*****

const n = 5;

for (let i = 0; i <= n; i++) {
    let row = '';

    for (let j = 0; j <= i; j++) {
        if (j !== 0) {
            row += '*';
        } else {
            row += i;
        }
    }

    console.log(row);
}

File: patterns/python-patterns/pattern64.py
# Pattern 64: ...

# 0
# 1*
# 2**
# 3***
# 4****
# 5*****

n = 5

for i in range(n + 1):
    for j in range(i + 1):
        if j != 0:
            print("*", end="")
        else:
            print(i, end="")
    print()
